[2023-06-25T02:23:59.161+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.inaccurate manual__2023-06-25T02:23:35.238861+00:00 [queued]>
[2023-06-25T02:23:59.202+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.inaccurate manual__2023-06-25T02:23:35.238861+00:00 [queued]>
[2023-06-25T02:23:59.206+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-25T02:23:59.258+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): inaccurate> on 2023-06-25 02:23:35.238861+00:00
[2023-06-25T02:23:59.285+0000] {standard_task_runner.py:57} INFO - Started process 604 to run task
[2023-06-25T02:23:59.306+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'manual__2023-06-25T02:23:35.238861+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpbwat0xm8']
[2023-06-25T02:23:59.321+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask inaccurate
[2023-06-25T02:23:59.498+0000] {task_command.py:410} INFO - Running <TaskInstance: my_dag.inaccurate manual__2023-06-25T02:23:35.238861+00:00 [running]> on host 8b092f32fb5b
[2023-06-25T02:23:59.776+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2023-06-25T02:23:35.238861+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-25T02:23:35.238861+00:00'
[2023-06-25T02:23:59.781+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-06-25T02:23:59.784+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo inaccurate']
[2023-06-25T02:23:59.820+0000] {subprocess.py:86} INFO - Output:
[2023-06-25T02:23:59.830+0000] {subprocess.py:93} INFO - inaccurate
[2023-06-25T02:23:59.836+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-25T02:23:59.893+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20230625T022335, start_date=20230625T022359, end_date=20230625T022359
[2023-06-25T02:23:59.970+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-25T02:24:00.014+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
